src/app/rpmostree-builtin-ex.cxx

```diff
@@ -26,6 +26,9 @@ static RpmOstreeCommand ex_subcommands[] = {
   { "livefs", (RpmOstreeBuiltinFlags)(RPM_OSTREE_BUILTIN_FLAG_HIDDEN),
     "Apply pending deployment changes to booted deployment",
     rpmostree_ex_builtin_apply_live },
+  { "search", (RpmOstreeBuiltinFlags)(RPM_OSTREE_BUILTIN_FLAG_HIDDEN),
+    "Fetch for the lastest version of a package from enabled repos",
+    rpmostree_builtin_override },
```

Comment:
I think this needs to be `rpmostree_ex_builtin_search` or so, and you'd need to define that function too.

Reply:
Or perhaps easier, you can just use e.g. `gdbus` to call the DBus method without writing custom code, see e.g. https://github.com/coreos/rpm-ostree/blob/fe152203aa31340c2c32f36aadf2a137ecec3873/tests/vmcheck/test-cached-rpm-diffs.sh#L62